src/main/java/com/visenze/visearch/internal/json/ImageResultMixin.java

```diff
@@ -18,6 +20,12 @@
     @JsonProperty("vs_value_map")
     protected Map<String, Object> vsMetadata;
 
+    @JsonProperty("alternatives")
+    protected List<ImageResult> alternatives;
+
+    @JsonProperty("tags")
+    protected Map<String, Object> tags;
```

Comment:
this is not consistent with ImageResult where it is <String, String>

Reply:
Align with the last update of ImageResult, so this part is not changed.